src/debug.c

```diff
@@ -63,42 +63,71 @@ void InstallBacktraceHandlers(void)
 typedef void (*VoidFunc)(void);
 VoidFunc debug_func_pointers[] = {
     (VoidFunc)ActiveGAPState,
+    (VoidFunc)ADDR_INT,
     (VoidFunc)ADDR_OBJ,
+    (VoidFunc)ADDR_PERM2,
+    (VoidFunc)ADDR_PERM4,
     (VoidFunc)ADDR_PPERM2,
     (VoidFunc)ADDR_PPERM4,
     (VoidFunc)ADDR_TRANS2,
     (VoidFunc)ADDR_TRANS4,
+    (VoidFunc)AINV_MUT,
+    (VoidFunc)AINV,
     (VoidFunc)AlwaysNo,
     (VoidFunc)AlwaysYes,
     (VoidFunc)ARE_INTOBJS,
     (VoidFunc)ASS_LIST,
+    (VoidFunc)ASS_LVAR_WITH_CONTEXT,
+    (VoidFunc)ASS_LVAR,
+    (VoidFunc)ASS_REC,
     (VoidFunc)AssConstantGVar,
     (VoidFunc)AssReadOnlyGVar,
     (VoidFunc)ASSS_LIST,
     (VoidFunc)BAG_HEADER,
     (VoidFunc)BASE_PTR_PLIST,
+    (VoidFunc)BLOCK_ELM_BLIST_PTR,
     (VoidFunc)BLOCKS_BLIST,
     (VoidFunc)BODY_FUNC,
     (VoidFunc)BODY_HEADER,
+    (VoidFunc)C_MAKE_INTEGER_BAG,
     (VoidFunc)C_MAKE_MED_INT,
     (VoidFunc)C_NORMALIZE_64BIT,
     (VoidFunc)C_SET_LIMB4,
     (VoidFunc)C_SET_LIMB8,
+    (VoidFunc)CACHE_OPER,
     (VoidFunc)CAPACITY_PLIST,
     (VoidFunc)CAPACITY_PREC,
     (VoidFunc)CHANGED_BAG,
+    (VoidFunc)CHAR_SINT,
     (VoidFunc)CHAR_VALUE,
     (VoidFunc)CHARS_STRING,
-    (VoidFunc)CheckRecursionBefore,
```

Comment:
Are you sure about this? This is still a `static inline void` in the stable branch